### DISCUSSION TITLE
How to join two streams?

### DISCUSSION THREAD

**Discussion by padzikm** [apache/pulsar#22216] [Q&A] [answered]
Hi, in apache kafka streams I can join streams - how can i do that in pulsar?

**Comment by lhotari** [comment, marked as answer]
Flink can be used for stream processing use cases with Pulsar. Flink contains a Pulsar connector: https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/pulsar/

For simple stream processing use cases, [Pulsar Functions](https://pulsar.apache.org/docs/next/functions-overview/) are available in Pulsar.
